README.md

```diff
@@ -98,13 +98,14 @@ The following are our draft proposals for the interface, written in Web IDL.
 
 ### Filesystem calls
 
-All filesystem calls are accessible through a global NativeIO instance.
+All filesystem calls are accessible through a global Storage Foundation
+instance.
 
 
 ```webidl
 // IMPORTANT: filenames are restricted to lower-case alphanumeric and
 // underscore (a-z, 0-9, _).
-interface NativeIOManager {
+interface NativeIOFileManager {
```

Comment:
```suggestion
interface StorageFoundationFileManager {
```

Can this be renamed, too?

Reply:
Good catch, I'll rename this soon - first in Chromium and, as soon as that's landed, in the explainer. Unfortunately,  `NativeIOFileManager` is still visible in the current version (but only if you look at it very closely in DevTools).